circle.yml

```diff
@@ -1294,6 +1307,44 @@ jobs:
       - run-driver-integration-tests:
           browser: electron
 
+  driver-integration-tests-chrome-experimentalSessionAndOrigin:
+    <<: *defaults
+    resource_class: medium
+    parallelism: 5
+    steps:
+      - run-driver-integration-tests:
+          browser: chrome
+          install-chrome-channel: stable
+          experimentalSessionAndOrigin: true
+
+  driver-integration-tests-chrome-beta-experimentalSessionAndOrigin:
+    <<: *defaults
+    resource_class: medium
+    parallelism: 5
+    steps:
+      - run-driver-integration-tests:
+          browser: chrome:beta
+          install-chrome-channel: beta
+          experimentalSessionAndOrigin: true
+
+  driver-integration-tests-firefox-experimentalSessionAndOrigin:
+    <<: *defaults
+    resource_class: medium
+    parallelism: 5
```

Comment:
Should we bump this to 6 or 7 since so many tests for multi-domain since we were concerned with longer CI times?

Reply:
It's hard to get a direct comparison but with current parallelization the CI job seems to run for roughly the same duration as our existing job. So I don't think we need to bump these right now.